cmd/krel/cmd/ff.go

```diff
@@ -70,7 +71,7 @@ func runFf(opts *ffOptions) error {
 	logrus.Infof("Preparing to fast-forward master@%s onto the %s branch", masterRef, branch)
 	repo, err := kgit.CloneOrOpenDefaultGitHubRepoSSH(rootOpts.repoPath, opts.org)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "unable to open/clone repo: %v", rootOpts.repoPath)
```

Comment:
Maybe put this message in the main `CloneOrOpen` function instead?

Reply:
Fixed inside the `git` package.